data/meson.build

```diff
@@ -0,0 +1,32 @@
+install_data(
+    meson.project_name() + '.appdata.xml',
+    install_dir: join_paths(get_option('prefix'), get_option('datadir'), 'metainfo')
```

Comment:
Customarily, you set `*dir` variables in top-level `meson.build` to avoid repetition.

Reply:
Okay ...

Reply:
The metainfo directory is only used once, in the data subdirectory. I see no need to define it globally.